molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java

```diff
@@ -23,7 +23,8 @@ public boolean isFinalState()
 		@Override
 		public EnumSet<JobStatus> getStateTransitions()
 		{
-			return EnumSet.of(SCHEDULED, RUNNING);
+			//FIXME temp
+			return EnumSet.of(SCHEDULED, RUNNING, FAILED, COMPLETED, CANCELLED);
```

Comment:
status transition from RUNNING to CREATED is not allowed. Allowed transitions [COMPLETED, FAILED, CANCELLED]